cfme/tests/containers/test_containers_summary.py

```diff
@@ -43,7 +43,6 @@ def test_containers_summary_objects(provider):
     prov_ui_values = dict()
     for obj_type in container_object:
         prov_ui_values[obj_type] = getattr(provider.summary.relationships, obj_type).value
-    sel.force_navigate('container_dashboard')
-    time.sleep(2)
-    for obj_type in container_object:
+        sel.force_navigate('container_dashboard')
```

Comment:
@pavelzag regarding your email and the intermittent 500 repsonse failures that you were seeing, that issue is being worked, and isn't related to your changes here.

I see that you're trying to collapse the two loops over container_object, but I'm not understanding the need to navigate multiple times? It looks like the actual test behavior is unchanged here. If the getattr() call assumes that you're already on the containers dashboard, the navigation should be done before that call.

To me it looks like it is just now needlessly navigating each loop, instead of gathering all the data and then going to the dashboard once.